pkg/policy/api/selector.go

```diff
@@ -185,6 +202,17 @@ func NewESFromLabels(lbls ...labels.Label) EndpointSelector {
 	return NewESFromMatchRequirements(ml, nil)
 }
 
+// NewESFromLabelsPtr creates a new endpoint selector from the given labels.
+func NewESFromLabelsPtr(lbls ...labels.Label) *EndpointSelector {
+	ml := map[string]string{}
+	for _, lbl := range lbls {
+		ml[lbl.GetExtendedKey()] = lbl.Value
+	}
+
+	esMatchReqs := NewESFromMatchRequirements(ml, nil)
+	return &esMatchReqs
```

Comment:
Would this work to avoid code duplication:
```
func NewESFromLabelsPtr(lbls []labels.Label) *EndpointSelector {
	return &NewESFromLabels(lbls)
}
```